tests/models/test_path.py

```diff
@@ -43,18 +53,28 @@ def test_status_case_3(self):
         self.assertEqual(current_path.status, EntityStatus.DISABLED)
 
     # This method will be used by the mock to replace requests.get
-    def _mocked_requests_get_status_case_4(*args, **kwargs):
-        return MockResponse({'key':'OK'}, 200)
-
-    @patch('requests.get', side_effect=_mocked_requests_get_status_case_4)
+    @classmethod
+    def _mocked_requests_get_status_case_4(cls, url):
+        endpoint = '%s/%s' % (settings.TOPOLOGY_URL, 'links')
+        if url != endpoint:
+            raise ValueError('Not a topology URL')
+
+        return MockResponse({'links': [
+            {'active': True},
+            {'active': True}
+        ]}, 200)
```

Comment:
I didn't understand this change. Is it really related to linting?

Reply:
I reverted the commit. It should be fixed in PR 119